web/src/main/webapp/ui/menu.ts

```diff
@@ -109,10 +128,27 @@ abstract class BaseMenu<MI extends BaseMenuItem> implements IHtmlElement {
         return this.cells[index];
     }
 
+    public expandMenu(parentMenu: string) {
+        const index = this.find(parentMenu);
+        this.selectedParentMenu = index;
+        if (index < 0) throw new Error("Cannot find menu item " + parentMenu);
+        this.subMenuCells[index].cells.forEach((cell) => {
+            cell.style.display = "table-cell";
+        });
+    }
+
     public addItem(mi: MI, enabled: boolean): HTMLTableDataCellElement {
         const index = this.items.length;
         this.items.push(mi);
         const trow = this.tableBody.insertRow();
+
+        const dummySubMenu = trow.insertCell(0);
+        dummySubMenu.classList.add("dummyMenu");
+        dummySubMenu.onmouseenter = () => this.hide();
+        this.subMenuCells[index] = initSubMenu();
+        this.subMenuCells[index].dummySubMenu = dummySubMenu;
```

Comment:
I added dummySubMenus as mouseExit listener in the second column. This simplify the implementation and more robust than the previous approach.
![image](https://user-images.githubusercontent.com/7077197/88850048-68beb780-d1b0-11ea-8b8f-1a92fc934241.png)